### DISCUSSION TITLE
See What Patches Are Available

### DISCUSSION THREAD

**Discussion by nochlab1** [ReVanced/revanced-discussions#288] [Question] [answered]
Hi is it possible to put a log of what patches are available as all I see in the progress channel in the discord is that now there are 21 patches but when i saw it was 18 so how do i know what the other 3 are so maybe make another channel or put it in the progress channel
thanks

**Comment by TheJeterLP** [comment, marked as answer]
You can use the -l command with the cli to list all available patches

**Reply by nochlab1** [reply]
Ye but that means getting my computer and having to download it and run it instead of simply seeing what was added

**Reply by TheJeterLP** [reply]
That is correct. That's why ReVanced is targeted towards developers for now :)

**Reply by nochlab1** [reply]
I don't see the connection i know what to do just would like to know if it's with the effort of going to my computer .... if it's a patch i don't need

**Reply by TheJeterLP** [reply]
You have to go to your computer to build and patch it anyways

**Reply by TheJeterLP** [reply]
But I will discuss it with the team if we add this

**Reply by nochlab1** [reply]
Ye but if i don't need the patch then i won't rebuild it!
Ok thanks hopefully the vote is in my favor

**Reply by TheJeterLP** [reply]
You should rebuild it often just for Bugfixes too

**Comment by TheJeterLP** [comment]
@oSumAtrIX @Sculas what's your opinion on this?

**Comment by quyleanh** [comment]
The full command is: `java -jar revanced-cli.jar -l -b revanced-patches.jar`

**Comment by oSumAtrIX** [comment]
You are currently not supposed to use ReVanced. It is mainly targeting development which is why everything is oriented towards devs. When we consider the end user, everything will be made the way you will expect it, updates, change logs, announcements and so on.

**Comment by nochlab1** [comment]
Thanks so much for adding the readme of patches helps a lot!

**Reply by TheJeterLP** [reply]
No problem :)
